src/EnergyPlus/DXCoils.cc

```diff
@@ -6883,13 +6867,13 @@ void InitDXCoil(EnergyPlusData &state, int const DXCoilNum) // number of the cur
 
             // calculate coil model at rating point
             if (thisDXCoil.DXCoilType_Num == HVAC::CoilDX_CoolingSingleSpeed) {
-                CalcDoe2DXCoil(state, DXCoilNum, HVAC::CompressorOperation::On, false, 1.0, 1.0, _, 1.0);
+                CalcDoe2DXCoil(state, DXCoilNum, HVAC::CompressorOp::On, false, 1.0, HVAC::FanOp::Cycling, _, 1.0);
```

Comment:
`1.0` is `FanOp::Cycling`. Raise your hand if you knew that.

Reply:
![image](https://github.com/NREL/EnergyPlus/assets/5778156/b00df0f0-9e0d-443e-8ff3-577929521f88)